tests/cli_terraform_test.go

```diff
@@ -172,3 +173,56 @@ func runTerraformCleanCommand(t *testing.T, binaryPath string, args ...string) {
 		t.Fatalf("Failed to run terraform clean: %v", stderr.String())
 	}
 }
+
+func TestCLITerraformENV(t *testing.T) {
```

Comment:
If we use this pattern, we do not increase the test coverage. I don’t know that it’s any better than using the YAML. The CLI test methods operate on the pre compiled binary.

Reply:
Please rewrite/refactor so we test the functions not by running smoke tests